Replay/Replay/ViewController.swift

```diff
@@ -85,44 +87,54 @@ class ViewController: UIViewController {
     }
     
     func initPlayer() -> Void {
-        
-        guard let url = URL(string: media) else {
-            playbackStatus = .unplayable
-            self.statusLabel?.text = "No media specified for playback"
-            return
+        var mediaArray = [URL]()
+        for media in urlArray {
```

Comment:
Lets construct the asset url directly from URL array. This way we dont need for loop for mediaArray.